libs/ledger-live-common/src/exchange/swap/hooks/useUpdateMaxAmount.ts

```diff
@@ -32,16 +32,21 @@ export const useUpdateMaxAmount = ({
   const [isMaxLoading, setIsMaxLoading] = useState(false);
 
   const toggleMax: SwapTransactionType["toggleMax"] = useCallback(
-    () =>
+    account =>
       setIsMaxEnabled(previous => {
         const next = !previous;
         if (previous) {
           setFromAmount(ZERO);
           setIsMaxLoading(false);
         }
+        let additionalFees;
+        if (account?.type === "Account" && account?.unit?.code === "ETH") {
+          additionalFees = new BigNumber(10000000000000000); // 0,01 ETHs
+        }
```

Comment:
Could we use more descriptive with the variable name (although I am not sure my name is the best one)?
I think it can be better create something more generic...tomorrow I assume we will add other families here 

```suggestion
        const familySecurityFee = {
                 ETH: 10000000000000000 // 0,01 ETHs
        }
         bridge.updateTransaction(tx => ({
          ...tx,
          useAllAmount: next,
          securityFee: account?.type === "Account" && familySecurityFee[account?.unit?.code],
        }));
```

Reply:
Idea was to reuse an existing field for ETH as a quick fix, so I'm not in favor or renaming it. Especially since the modification made is generic, it is just more fees deducted, not specific to a use case. More info on [slack](https://ledger.slack.com/archives/C02LACPH3U3/p1709033172133369)

Also I don't think this fix will be extended upon more coins. 
Max button logic will be removed with swap-live-app to use a better approach of treating max button as user input to not have two different logics

Reply:
Follow my understood this is the solution that we want to apply in the web app and migrate LLM will take months...
What is the the difference between "evm" and other chains? If the fix works, I am sure we will e applying it for other coins. 
On the other hand can we change "additionalFees" for "marginFee" or marginEthFee"?

Reply:
You are right updating it to include all evm chains
but don't want to rename `additionalFees` it is generic like this